polymer-autohost/src/main/java/eu/pb4/polymer/autohost/impl/providers/StandaloneWebServerProvider.java

```diff
@@ -89,6 +90,8 @@ public void handle(HttpExchange exchange) throws IOException {
                 ) {
                     exchange.getResponseHeaders().add("Server", "polymer-autohost");
                     exchange.getResponseHeaders().add("Content-Type", "application/zip");
+                    if (PolymerResourcePackMod.sha1 != null)
```

Comment:
Don't think this would effect how vanilla handles it, only if using some proxy witch cache I guess?